src/scripts/transformRecordedTests/upgradeDecorations.ts

```diff
@@ -0,0 +1,88 @@
+import { TestCaseFixture } from "../../testUtil/TestCaseFixture";
```

Comment:
we could probably remove this, but it is kinda useful to have one sitting around, even if we also have an example in the docs 🤷